src/hrc721.ts

```diff
@@ -2,7 +2,7 @@ import { Contract } from '@harmony-js/contract'
 import { Harmony } from '@harmony-js/core'
 import { BigNumber, logger } from 'ethers'
 import { Logger } from 'ethers/lib/utils'
-import { BaseHRC721 } from './base-hrc721'
+import { BaseHRC721 } from './base-HRC721'
```

Comment:
why the name change? I think we are using `base-hrc721`

Reply:
It's because I changed the name (with uppercase only) 😅